tests/objc/Example.m

```diff
@@ -241,6 +241,36 @@ +(NSUInteger) overloaded:(NSUInteger)arg1 extraArg:(NSUInteger)arg2
     return arg1 + arg2;
 }
 
++(NSUInteger) overloaded:(NSUInteger)arg extraArg1:(NSUInteger)arg1 extraArg2:(NSUInteger)arg2
+{
+    return 1;
+}
+
++(NSUInteger) overloaded:(NSUInteger)arg extraArg2:(NSUInteger)arg2 extraArg1:(NSUInteger)arg1
+{
+    return 2;
+}
+
++(NSUInteger) overloaded:(NSUInteger)arg orderedArg1:(NSUInteger)arg1 orderedArg2:(NSUInteger)arg2
+{
+    return 0;
+}
+
++(NSUInteger) overloaded:(NSUInteger)arg duplicateArg:(NSUInteger)arg1 duplicateArg:(NSUInteger)arg2
+{
+    return arg1 + arg2;
+}
+
++(void) dummyForException:(NSUInteger)arg
+{
+    // Dummy method for testing exception
+}
```

Comment:
There's no need to have these extra methods just to demonstrate we can't call them correctly. The test will work just as well by calling `Example.overloaded(0, notAnArgument=10)`. It would be arguably *better* to use `overloaded:...` , because there are more options, and some of them are ambiguous.